### PR TITLE
YOUR_USERNAME adds Orlando file

### LINKED ISSUE #2: YOUR_USERNAME practices using GitHub Flow
- [x] Create a branch off of `master`
- [x] Add some commits
- [x] Open a pull request
- [x] Collaborate ⭐️ 
- [x] Merge when we get at least 1 👍

### PR DESCRIPTION
**Pull request by githubteacher**
The purpose of this PR is to add the Orlando, FL file to the hometown files. 

This PR closes #2

### CODE REVIEW

File: YOUR_USERNAME-orlando.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Orlando, FL
+
+### Restaurants to Visit
+- The Pharmacy
```
Comment (githubteacher) [inline YOUR_USERNAME-orlando.md:4]: This is a *food* place!? @githubteacher

File: YOUR_USERNAME-orlando.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Rocco's Tacos
+
+### Natural Places
+- Payne's Praire is a drive way, but a beautiful place
+- Forever Florida Eco Park
+
+### Man-made Places
+- Gatorland
+- Disney World
```
Comment (dan-bock) [inline YOUR_USERNAME-orlando.md:14]: @githubteacher Maybe you should add your favorite ride at Disneyworld!!

File: YOUR_USERNAME-orlando.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+### Restaurants to Visit
+- The Pharmacy
+- Rocco's Tacos
+
+### Natural Places
+- Payne's Praire is a drive way, but a beautiful place
+- Forever Florida Eco Park
+
+### Man-made Places
```
Comment (scott-larson) [inline YOUR_USERNAME-orlando.md:11]: Are the Gators "man-made"?

**Comment by githubteacher** [comment]
@scott-larson 😆 did I make it a little better in d6e3197

@matt-roeckers check out commit ad586b9

**Review by dan-bock — APPROVED** [review]
That "ride" at EPCOT sounds fun.  Approved!

**Review by jeffrey-jakubowsky — CHANGES_REQUESTED** [review]
What is your favorite show?

**Pull request merged**